_scripts/build-storybooks.sh

```diff
@@ -105,13 +105,16 @@ if [[ ! -z $CI_PULL_REQUEST ]]; then
     PR_NUMBER=$(echo $CI_PULL_REQUEST | cut -d/ -f7);
     PR_URL="$STORYBOOKS_URL/pulls/$PR_NUMBER/";
     COMMIT_URL="$STORYBOOKS_URL/commits/$COMMIT_HASH/";
+
     curl \
         -H'Content-Type: application/json' \
         -H"Authorization: token $STORYBOOKS_GITHUB_TOKEN" \
-        --data "{\"body\":\"# Storybook Deployment\\nPull Request: $PR_URL\\nCommit: $COMMIT_URL\"}" \
-        https://api.github.com/repos/${PROJECT_REPO}/issues/${PR_NUMBER}/comments;
+        --data "{\"state\":\"success\",\"target_url\":\"$PR_URL\",\"context\":\"storybooks: pull request\",\"description\":\"Storybook deployment for Pull Request\"}" \
+        https://api.github.com/repos/${PROJECT_REPO}/statuses/${COMMIT_HASH};
 fi
 
 echo "Cleaning up $PUBLISH_ROOT"
 cd ..
 rm -rf $PUBLISH_ROOT
+
+0d2a28b2412479f9e8fb07f8a1a2666784957f3a
```

Comment:
What's this doing?

Reply:
Uhh, me being dumb is what that's doing 😅